src/full/Agda/Interaction/Highlighting/LaTeX.hs

```diff
@@ -609,12 +609,10 @@ generateLaTeX i = do
 
   options <- TCM.commandLineOptions
 
-  dir <- case O.optGHCiInteraction options of
-    False -> return $ O.optLaTeXDir options
-    True  -> do
-      sourceFile <- Find.srcFilePath <$> Find.findFile mod
-      return $ filePath (projectRoot sourceFile mod)
-                 </> O.optLaTeXDir options
+  dir <- if O.optGHCiInteraction options then (do
```

Comment:
The parentheses here are quite horrible, I prefer the original version. A multiway `if` would be even better here.